ccp/modules/ovis-compose.yml

```diff
@@ -1,4 +1,134 @@
+      - FHIR_SERVER_URL=${FHIR_SERVER_URL:-http://bridgehead-ccp-blaze:8080/fhir}
+      - FHIR_USERNAME=${FHIR_USERNAME}
+      - FHIR_PASSWORD=${FHIR_PASSWORD}
+    volumes:
+      - /var/cache/bridgehead/ccp/ovis/shared_data:/app/output
+
+  mongo:
+    image: mongo:${MONGO_VER:-latest}
+    restart: always
+    command: mongod
+    volumes:
+      - /var/cache/bridgehead/ccp/ovis/mongo/mongo_db:/data/db
+      - /var/cache/bridgehead/ccp/ovis/mongo/mongo_conf:/data/configdb
+      - /var/cache/bridgehead/ccp/ovis/mongo/init/init.js:/docker-entrypoint-initdb.d/init.js
+
+  init_onco:
+    image: docker.verbis.dkfz.de/ovis/adt-mon-gql-backend:latest
+    restart: on-failure
+    working_dir: /app
+    user: root
+    entrypoint: >
+      sh -c "
+        while [ ! -f /shared/omock.json ]; do
+          echo 'Waiting for omock.json...'
+          sleep 5
+        done;
+        mkdir -p ./prep &&
+        cp /shared/omock.json ./prep/omock.json &&
+        node ./mgDB/prep/preprocessor.mjs;
+        echo 'Processing complete';
+        tail -f /dev/null"
+    depends_on:
+      - mongo
+      - fhir-transformer
+    environment:
+      - APOLLO_PORT=${APOLLO_PORT:-4001}
+      - CREDOS_PORT=${CREDOS_PORT:-4000}
+      - MONGO_VER=latest
+      - DB=${DB:-onc_test}
+      - ADRESS=${ADRESS:-mongodb://mongo:27017}
+    volumes:
+      - /var/cache/bridgehead/ccp/ovis/shared_data:/shared
+      - /var/cache/bridgehead/ccp/ovis/init_onco/node_modules:/app/node_modules  
+
+  backend:
+    image: docker.verbis.dkfz.de/ovis/adt-mon-gql-backend:latest
+    restart: always
+    working_dir: /app
+    environment:
+      - APOLLO_PORT=${APOLLO_PORT:-4001}
+      - CREDOS_PORT=${CREDOS_PORT:-4000}
+      - MONGO_VER=latest
+      - NODE_ENV=development
+      - CORS_ORIGIN=*
+      - DB=${DB:-onc_test}
+      - ADRESS=${ADRESS:-mongodb://mongo:27017}
+    depends_on:
+      - mongo
+      - init_onco
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:${APOLLO_PORT:-4001}/health"]
+      interval: 5s
+      timeout: 3s
+      retries: 5
+      start_period: 10s
+    entrypoint: ["node", "--watch", "index.js"]
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.ovis-backend.tls=true"
+      - "traefik.http.services.ovis-backend.loadbalancer.server.port=4001"
+
+  frontend:
+    image: docker.verbis.dkfz.de/ovis/adt-mon-gql-frontend:latest
+    restart: always
+    environment:
+      - NODE_ENV=development
+      - VITE_GRAPHQL_URL=http://localhost:${APOLLO_PORT:-4001}/graphql
+    depends_on:
+      backend:
+        condition: service_healthy
+    working_dir: /app
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.ovis-frontend.tls=true"
+      - "traefik.http.routers.ovis-frontend.rule=Host(`${GUI_HOST:-localhost}`)"
+      # - "traefik.http.routers.lens.middlewares=traefik-forward-auth"
+      - "traefik.http.services.ovis-frontend.loadbalancer.server.port=5173"
+    volumes:
```

Comment:
What purpose does this volume serve? As far as i know, /app/.svelte-kit contains the application (frontend and backend) and besides using docker watch for faster development iterations i don't see any benefit in having this.